routes/golfer_search.go

```diff
@@ -0,0 +1,64 @@
+package routes
+
+import (
+	"net/http"
+
+	"github.com/code-golf/code-golf/config"
+	"github.com/code-golf/code-golf/session"
+)
+
+type Solution struct {
+	Code    string
```

Comment:
This will cause some churn in your typescript but I think JSON should have lowercase keys, so add stuff like:
```go
type Solution struct {
	Code    string `json:"code"`
	...
```